examples/xlio_ultra_api_ping_pong.c

```diff
@@ -61,20 +95,15 @@ struct app_state {
 
     /* RTT measurement */
     struct timespec ping_sent_time; /* Timestamp when last ping was sent */
-
-    /* TX zero-copy support */
-    struct ibv_pd *pd;
-    struct ibv_mr *mr_buf;
-    char sndbuf[256];
 };
 
 /* Default configuration */
 static struct app_config g_config = {.is_server = false,
```

Comment:
comment on all. we should be dummy-proof
suggestion:
`static struct app_config g_config = {
    .is_server = false,
    .ip = NULL,             /* Server IP (required for client mode) */
    .client_ip = NULL,      /* Optional: local IP to bind client socket */
    .port = 8080,           /* Default TCP port */
    .ping_count = 3,        /* Number of ping messages to exchange */
    .timeout_seconds = 0    /* 0 = no timeout */
};`

Reply:
done